transports/azure-storage-queues/multi-storageaccount-support_message_endpoint_mapping_asq[,7].partial.md

```diff
@@ -0,0 +1,15 @@
+### Message endpoint mappings
```

Comment:
The filename is off.  Shouldn't the range be `(,7]`? I.e. from any version up to 7 (included).